src/adapter/src/coord/timestamp_oracle/postgres_oracle.rs

```diff
@@ -145,44 +142,268 @@ impl PostgresTimestampOracleConfig {
             }
         };
 
+        let dynamic = DynamicConfig {
```

Comment:
optional: should these reuse the `DEFAULT_` constants? I lean weakly toward yes. maybe `impl Default for DynamicConfig` and then use it both here and in the real `new` impl